plugins/org.eclipse.xtext.generator/src/org/eclipse/xtext/generator/parser/antlr/AbstractAntlrGeneratorFragment.java

```diff
@@ -92,23 +92,7 @@ public void addAntlrParam(String param) {
 		String[] result = params.toArray(new String[params.size()]);
 		return result;
 	}
-	
-	private AntlrCodeQualityHelper codeQualityHelper = new AntlrCodeQualityHelper();
```

Comment:
I wanted this to be customizable from mwe without the need to override all Antlr fragments. Even though the old infrastructure will eventually be deprecated, we should not make customizing more painful than necessary.